taxi/tests/test_model.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelsTests(TestCase):
+    def test_manufacturer_str(self) -> None:
+        manufacturer = Manufacturer.objects.create(
+            name="TestName", country="TestCountry"
+        )
+        self.assertEqual(
+            str(manufacturer), f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_create_author_with_license_number(self) -> None:

```

Comment:
The method name 'test_create_author_with_license_number' seems to be incorrect. It should be 'test_create_driver_with_license_number' to accurately reflect the entity being tested.